chroniker/admin.py

```diff
@@ -1,20 +1,26 @@
 from django import forms
 from django.conf import settings
-from django.conf.urls import url
 from django.contrib import admin
 from django.core.management import get_commands
-from django.urls import reverse, NoReverseMatch
 from django.db import models
 from django.forms import TextInput
 from django.shortcuts import render
-from django.utils.encoding import force_text
+from django.urls import reverse, NoReverseMatch
+try:
+    from django.urls import re_path
+except ImportError:
+    from django.conf.urls import url as re_path
+from django.utils.encoding import force_str
```

Comment:
`force_str` was added in Django 1.5, so it seems unlikely that we still need a backward compatible import for `force_text`.

Reply:
Is the community off Python 2 yet? Might be worth polling but not sure how